bulk-import/ndjson-generator/NDJsonGenerator.csproj

```diff
@@ -0,0 +1,13 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>net6.0</TargetFramework>
```

Comment:
Nit: suggest double checking the convention here. A lot of the .NET SDK projects have moved to .NET 7 already. I know .NET 6.0 is the LTR, but .NET 7 comes with some performance improvements.